card.go

```diff
@@ -585,6 +617,8 @@ type CardFace struct {
 	// This will only be set if the card is not in English.
 	PrintedName *string `json:"printed_name"`
 
+	FlavorName *string `json:"flavor_name,omitempty"`
```

Comment:
I was looking through the documented card face object and I don't see this one there. I only see it on the card object. Is the documentation incorrect? Do you happen to have an example card that has a card face with a flavor name?

https://scryfall.com/docs/api/cards#card-face-objects

Reply:
Hmmm that one seems to not have multiple card faces so it only has the flavor_name on the card object and not on the card face object.